integrations/octopus/.port/resources/blueprints.json

```diff
@@ -0,0 +1,292 @@
+          "format": "date-time",
+          "description": "The datetime the release was assembled"
+        },
+        "channelId": {
+          "type": "string",
+          "title": "Channel ID",
+          "description": "The ID of the channel associated with the release"
+        },
+        "releaseNotes": {
+          "type": "string",
+          "title": "Release Notes",
+          "description": "Notes provided for the release"
+        },
+        "projectId": {
+          "type": "string",
+          "title": "Project ID",
+          "description": "The ID of the project this release belongs to"
+        },
+        "url": {
+          "type": "string",
+          "title": "Release URL",
+          "format": "url",
+          "description": "The URL to access the release in Octopus Deploy"
+        }
+      }
+    },
+    "calculationProperties": {},
+    "relations": {
+      "project": {
+        "target": "octopusDeployProject",
+        "title": "Project",
+        "description": "The project to which this release belongs",
+        "required": false,
+        "many": false
+      }
+    }
+  },
+  {
+    "identifier": "octopusDeployDeployment",
+    "title": "Octopus Deploy Deployment",
+    "icon": "Octopus",
+    "description": "A deployment in Octopus Deploy",
+    "schema": {
+      "properties": {
+        "releaseId": {
+          "type": "string",
+          "title": "Release ID",
+          "description": "The ID of the release being deployed"
+        },
+        "environmentId": {
+          "type": "string",
+          "title": "Environment ID",
+          "description": "The ID of the environment to which the deployment is targeted"
+        },
+        "created": {
+          "type": "string",
+          "title": "Created",
+          "format": "date-time",
+          "description": "The datetime when the deployment was created"
+        },
+        "deployedBy": {
+          "type": "string",
+          "title": "Deployed By",
+          "description": "The user or system that performed the deployment"
+        },
+        "taskId": {
+          "type": "string",
+          "title": "Task ID",
+          "description": "The ID of the task associated with the deployment"
+        },
+        "projectId": {
+          "type": "string",
+          "title": "Project ID",
+          "description": "The ID of the project associated with the deployment"
+        },
+        "failureEncountered": {
+          "type": "boolean",
+          "title": "Failure Encountered",
+          "description": "Indicates if any failure was encountered during the deployment"
+        },
+        "comments": {
+          "type": "string",
+          "title": "Comments",
+          "description": "Comments regarding the deployment"
+        },
+        "spaceId": {
+          "type": "string",
+          "title": "Space ID",
+          "description": "The ID of the space in which the deployment exists"
+        },

```

Comment:
Looks like space is central in Octopus. Can it be a new blueprint?

Reply:
Done